src/main/java/net/sf/jabref/logic/xmp/XMPUtil.java

```diff
@@ -626,11 +626,10 @@ private static void writeToDCSchema(XMPSchemaDublinCore dcSchema,
              */
             if ("author".equals(field)) {
                 String authors = resolvedEntry.getField(field);
-                AuthorList list = AuthorList.getAuthorList(authors);
+                AuthorList list = AuthorList.getAuthors(authors);
 
-                int n = list.size();
-                for (int i = 0; i < n; i++) {
-                    dcSchema.addCreator(list.getAuthor(i).getFirstLast(false));
```

Comment:
I think I introduced the second getAuthorList... Didn't think of it then...

Or getAuthorList().getListOfAuhtors() which in some sense is more
self-explanatory but maybe also open for more confusion. Your suggestion
works for me, so as long as they are different I'm OK with it.